cmd/osv-scanner/main.go

```diff
@@ -12,11 +12,23 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+var (
+	version = ""
```

Comment:
Can you add "`dev`" to version here as a default, and `n/a` for commit and date?

Reply:
Done!